scripts/settings-policy-set.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/..")"
+
+# Construct the proposal
+cp $REPO_ROOT/$SETTINGS_POLICY_PROPOSAL $WORKSPACE/proposals/set_settings_policy.json
```

Comment:
Would it be more flexible to allow settings policy path to not have to be relative to REPO_ROOT?

How about this? Then relative paths would still work for convenience 
```
(cd $REPO_ROOT && cp $SETTINGS_POLICY_PROPOSAL $WORKSPACE/proposals/set_settings_policy.json)
```